docs/API/LorisRESTAPI.md

```diff
@@ -0,0 +1,460 @@
+it will result in a 403 Forbidden response. PUT or POST requests used for the creation of resources
+do not require ETags.
+
+DELETE is not supported on any resource defined in this API.
+
+# 1.1 Authentication
+
+If a user is logged in to Loris and can be authenticated using the standard session mechanism,
+no further authentication is required. Requests will be evaluated as requests from that user,
+so that standard Loris modules can simply use the API.
+
+If a user is not logged in to Loris (for instance, in a third party app or a CORS application),
+they can be be authenticated using [JSON Web Tokens](https://jwt.io).
+
+The client should POST a request to /login with a payload of the form
+
+```js
+{
+    "username" : username,
+    "password" : password
+}
+```
+
+If the username and password are valid, the API will respond with a 200 OK and payload
+of the form
+
+```js
+{
+    "token" : /* JWT token */
+}
+```
+
+Otherwise, it will return a 401 Unauthorized response.
+
+If the token is returned, it should be included in an "Authorization: Bearer token" header
+for any future requests to authenciate the request.
+
+# 2.0 Project API
+
+The Project API lives under the /projects portion of the API URL hierarchy. It is used to get
+project specific settings or data. PUT and PATCH are not currently supported for the part of
+the API living under /projects.
+
+```
+GET /projects
+```
+
+Will return a list of projects in this Loris instance. There is no corresponding PUT or PATCH
+request. The JSON returned is of the form:
+
+```js
+{
+    "Projects" : {
+        "ProjectName1" : {
+            "useEDC" : boolean,
+            "PSCID" : PSCIDSettings
+        },
+        "ProjectName2" : {
+            "useEDC" : boolean,
+            "PSCID" : PSCIDSettings
+        },
+        ...
+}
+```
+
+If the Loris instance does not use projects, the API will return a single project called "loris"
+with the appropriate settings for the Loris instance.
+
+useEDC represents a boolean determining whether the EDC date should be included
+in candidates returned by the API.
+
+PSCID represents a JSON object with the configuration settings for PSCIDs in this
+project.
+
+It has the form:
+
+```js
+{
+    "Type" : "prompt|auto",
+    "Regex" : "/regex/"
+}
+```
+
+Where regex is a regular expression that can be used to validate a PSCID for this project.
+
+If the type is "prompt", the user should be prompted to enter the PSCID for new candidates.
+If the type is "auto", the server will automatically generate the PSCID.
+
+Note that sometimes in Loris configurations "Site" is a part of the PSCID. This will be
+denoted by the string "SITE{1,1}" inside of the regex returned. This string should be replaced
+by the 3 letter site alias before attempting to pass this regex to a regular expression parser
+or it will result in false negatives.
+
+
+```
+GET /projects/$ProjectName
```

Comment:
I tried to get /projects/loris and got this return 

```
{
  "error": {
    "error": "Invalid project"
  }
}
```
I get the loris project if i get just /projects